src/app/sidebar/Sidebar.tsx

```diff
@@ -5,7 +5,6 @@ import { SelectedCourse } from '../../App';
 import { Course, KualiSubject } from '../../fetchers';
```

Comment:
the import for `useState` should probably be removed (couldn't add a comment there)